src/main/java/com/genesys/workspace/VoiceApi.java

```diff
@@ -416,7 +451,7 @@ public void makeCall(String destination) throws WorkspaceApiException {
      */
     public void makeCall(
             String destination,
-            KeyValueCollection userData
+            Map<String,Object> userData
```

Comment:
Same comment for these - would prefer to keep KeyValueCollection exposed in public API.

Reply:
okay